README.md

```diff
@@ -57,15 +57,14 @@ Before running deployment configure `secure.py` settings in the
 
 ### Local deployment
 
-Local deployment can be started by the docker-compose up command in the
-console:
+Local deployment can be started by the make command in the
+console in `bridge_adaptivity` directory:
 
-    [sudo] docker-compose -f docker-compose_local.yml up
+    [sudo] make run-local
 
+Volume "pgs" is added to the the database container.
 
-  Volume "pgs" is added to the the database container.
-
-  Note: Development server available on `localhost:8008`
+  Note: Development server available on `localhost:8000`
```

Comment:
This change looks odd to me

Reply:
I deleted this change. Local Bridge run in the 8008 port.